test/pylib/s3_proxy.py

```
@@ -183,13 +183,13 @@ def process_request(self):
                 body = self.rfile.read(int(content_length))
 
             if policy.should_forward:
-                target_url = self.minio_uri + self.path
+                target_url = self.s3_uri + self.path
                 headers = {key: value for key, value in self.headers.items()}
                 try:
                     response = requests.request(self.command, target_url, headers=headers, data=body)
```

Ruff (0.16.3)

[error] Probable use of `requests` call without timeout

(S113)